### PR TITLE
feat(firestore-bundle-builder): migrate firestore-bundle-builder to official

### PR DESCRIPTION
**Pull request by Ehesp**

### CODE REVIEW

File: firestore-bundle-builder/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Version 0.1.2
```
Comment (Ehesp) [inline firestore-bundle-builder/CHANGELOG.md:1]: This file should probably be reset? Or are we keeping the previous versions?

File: firestore-bundle-builder/extension.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# Learn detailed information about the fields of an extension.yaml file in the docs
+
+name: firestore-bundle-builder
+specVersion: v1beta
+version: 0.1.0
```
Comment (Ehesp) [inline firestore-bundle-builder/extension.yaml:5]: What should this be?

**Comment by Ehesp** [comment]
@huangjeff5 Can you check the comments above re versions? Are we starting fresh (since it's a new extension name) or continuing from the old one?

**Comment by huangjeff5** [comment]
> @huangjeff5 Can you check the comments above re versions? Are we starting fresh (since it's a new extension name) or continuing from the old one?

Just copying here for posterity, let's start fresh.

File: firestore-bundle-builder/admin-dashboard/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Firestore Bundle Builder - Admin Dashboard
+
+This repository contains the source code for the Admin Dashboard for the Firestore Bundle Builder extension. Use this to locally create bundles.
+
+## Installation
+
+Clone this repository:
+
+```bash
+git clone ... todo
```
Comment (dackers86) [inline firestore-bundle-builder/admin-dashboard/README.md:10]: Todo?

Comment (Ehesp) [reply firestore-bundle-builder/admin-dashboard/README.md:10]: Ah yes, needs updating to point to this repo.

**Review by dackers86 — APPROVED** [review]

**Comment by dackers86** [comment]
Closing. Ported to https://github.com/firebase/firestore-bundle-builder

**Pull request closed without merging**